testdata/taskrouter_responses.go

```diff
@@ -84,3 +84,79 @@ var TaskQueueCreateResponse = []byte(`
     "workspace_sid": "WS7a2aa7d8acc191786ad3c647c5fc3110"
   }
 `)
+
+var WorkflowResponse = []byte(`
+{
+    "account_sid": "AC58f1e8f2b1c6b88ca90a012a4be0c279",
+    "assignment_callback_url": "http://example.com",
+    "configuration": "task-routing:\n  - filter: \n      - 1 == 1\n    target:\n      - queue: WQ63868a235fc1cf3987e6a2b67346273f\n        set-priority: 0\n",
+    "date_created": "2014-05-14T10:50:02Z",
+    "date_updated": "2014-05-14T23:26:06Z",
+    "document_content_type": "application/json",
+    "fallback_assignment_callback_url": null,
+    "friendly_name": "Default Fifo Workflow",
+    "sid": "WW63868a235fc1cf3987e6a2b67346273f",
+    "task_reservation_timeout": 120,
+    "url": "https://taskrouter.twilio.com/v1/Workspaces/WS58f1e8f2b1c6b88ca90a012a4be0c279/Workflows/WF63868a235fc1cf3987e6a2b67346273f",
+    "workspace_sid": "WS58f1e8f2b1c6b88ca90a012a4be0c279",
+    "links": {
+      "statistics": "https://taskrouter.twilio.com/v1/Workspaces/WS58f1e8f2b1c6b88ca90a012a4be0c279/Workflows/WF63868a235fc1cf3987e6a2b67346273f/Statistics",
+      "real_time_statistics": "https://taskrouter.twilio.com/v1/Workspaces/WS58f1e8f2b1c6b88ca90a012a4be0c279/Workflows/WF63868a235fc1cf3987e6a2b67346273f/RealTimeStatistics",
+      "cumulative_statistics": "https://taskrouter.twilio.com/v1/Workspaces/WS58f1e8f2b1c6b88ca90a012a4be0c279/Workflows/WF63868a235fc1cf3987e6a2b67346273f/CumulativeStatistics"
+    }
+  }
+`)
+
+var WorkflowCreateResponse = []byte(`
+{
+  "account_sid": "AC58f1e8f2b1c6b88ca90a012a4be0c279",
+  "assignment_callback_url": "http://example.com",
+  "configuration": {
```

Comment:
OK - the API is actually just returning JSON data in a string. The docs are wrong, so I'd suggest using HTTP to create/retrieve the resources and then copying here.

```json
    "configuration": "{\"task_routing\":{\"default_filter\":{\"queue\":\"WQ0c1274082082355320d8a41f94eb57aa\"}}}",
```

Reply:
Updated 👍